app/transactions/hyrax/transactions/iiif_print_container_decorator.rb

```diff
@@ -28,27 +28,29 @@ class IiifPrintContainerDecorator
         end
       end
 
-      namespace 'change_set' do |ops|
-        ops.register 'update_work' do
-          steps = Hyrax::Transactions::WorkUpdate::DEFAULT_STEPS.dup
-          steps.insert(steps.index('work_resource.update_work_members') + 1, 'work_resource.set_child_flag')
-          Hyrax::Transactions::WorkUpdate.new(steps: steps)
-        end
-      end
-
-      if defined?(Bulkrax::Container)
-        namespace 'work_resource' do |ops|
-          ops.register Bulkrax::Container::UPDATE_WITH_BULK_BEHAVIOR do
-            steps = Bulkrax::Container::UPDATE_WITH_BULK_BEHAVIOR_STEPS.dup
+      if ENV['HYKU_IIIF_PRINT']
```

Comment:
The knowledge of why we do or don't have `HYKU_IIIF_PRINT` is a bit odd.  I'm not excited to have a `ENV` from Hyku show up here.

What I'm wondering is if we should have Hyku decorate the `work_resource.set_child_flag` to account for it's implementation foibles relative to how Hyku is using the `HYKU_IIIF_PRINT`.

Put another, when we see this variable, my read is `HYKU_IIIF_PRINT == false` means don't do anything IIIF Print related.  But that is not what it means in Hyku, thus we should limit the knowledge/foibles of that ENV to Hyku only.

Reply:
TODO: In Hyku, remove this step based on that env var